code/modules/mod/mod_theme.dm

```diff
@@ -143,6 +143,8 @@
 	control_part_datum.part_item = mod
 	mod.mod_parts["[mod.slot_flags]"] = control_part_datum
 	for(var/path in variants[default_skin])
+		if(!ispath(path))
+			continue
```

Comment:
could you explain this?

Reply:
https://github.com/tgstation/tgstation/blob/57049f0a2d5686728c9030d4ed4dbc17ed766517/code/modules/mod/mod_theme.dm#L197-L201
Here, there's support for overloading the icon file for the entire modsuit. If you try and use that currently (seems to only be downstream), this runtimes since it isn't actually a typepath

Reply:
ugh, thats annoying... i think ill need to add some general category (maybe a list tied to /obj/item/mod/control?) with stuff like this.

Reply:
What does that mean for the PR? Should I drop that change pending your fix or is it workable for now?